scriptworker/constants.py

```diff
@@ -176,14 +176,16 @@
     # Map scopes to restricted-level
     'cot_restricted_scopes': frozendict({
         'firefox': frozendict({
-            'project:releng:balrog:release': 'release',
-            'project:releng:beetmover:release': 'release',
-            'project:releng:pushapk:release': 'release',
-            'project:releng:signing:cert:release-signing': 'release',
-            'project:releng:balrog:nightly': 'nightly',
-            'project:releng:beetmover:nightly': 'nightly',
-            'project:releng:pushapk:nightly': 'nightly',
-            'project:releng:signing:cert:nightly-signing': 'nightly',
+            'project:releng:balrog:release': 'all-release-branches',
+            'project:releng:beetmover:release': 'all-release-branches',
+            'project:releng:googleplay:release': 'release',
+            'project:releng:signing:cert:release-signing': 'all-release-branches',
+            'project:releng:googleplay:beta': 'beta',
+            'project:releng:googleplay:aurora': 'aurora',
+            'project:releng:balrog:nightly': 'all-nightly-branches',
+            'project:releng:beetmover:nightly': 'all-nightly-branches',
+            'project:releng:pushapk:nightly': 'all-nightly-branches',
```

Comment:
Should we keep this scope, now that we only rely on `project:releng:googleplay:*`?